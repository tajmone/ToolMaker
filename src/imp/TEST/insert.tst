%%({1,2,3,4,5}[0])
%%({1,2,3,4,5}[1])
%%({1,2,3,4,5}[2])
%%({1,2,3,4,5}[3])
%%({1,2,3,4,5}[4])
%%({1,2,3,4,5}[5])
%%({1,2,3,4,5}[6])
%%({1,2,3,4,5}[7])
