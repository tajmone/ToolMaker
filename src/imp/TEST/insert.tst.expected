########## insert ##########
%%({1,2,3,4,5}[0])
%%({1,2,3,4,5}[1])
%%({1,2,3,4,5}[2])
%%({1,2,3,4,5}[3])
%%({1,2,3,4,5}[4])
%%({1,2,3,4,5}[5])
%%({1,2,3,4,5}[6])
%%({1,2,3,4,5}[7])
########## ../imp insert.tst >insert.tst.tmp; cat insert.tst.tmp; rm insert.tst.tmp ##########


insert.tst

    7.  %%({1,2,3,4,5}[6])
=====>                 1

  *1*   209 E : Index value "6" out of range.

    8.  %%({1,2,3,4,5}[7])
=====>                 1

  *1*   209 E : Index value "7" out of range.


2 error(s) detected.
No warnings issued.

5
1
2
3
4
5
%%({1,2,3,4,5}[6])
%%({1,2,3,4,5}[7])
